collectors/freeipmi.plugin/freeipmi_plugin.c

```diff
@@ -1833,6 +1835,20 @@ int main (int argc, char **argv) {
     for(iteration = 0; 1 ; iteration++) {
         usec_t dt = heartbeat_next(&hb, step);
 
+        if (create_avail_chart) {
```

Comment:
Hello @ilyam8 ,

You do not need the variable `create_avail_chart`, you can check if `iteration` is equal zero and write to `stdout`.

Best regards!